sbi/density_estimators/base.py

```diff
@@ -0,0 +1,58 @@
+from pyknos.nflows.flows import Flow
+import torch
+from torch import Tensor, nn
+from typing import Optional,Union
+from warnings import warn
+
+class DensityEstimator:
+    r"""Base class for density estimators.
+    The density estimator class is a wrapper around neural networks that 
+    allows to evaluate the log_probs, sample, and provide the loss of $theta,x$ pairs.
```

Comment:
please put log_prob, sample, and loss in backticks such that it is clear that these are the methods that it implements.